src/pip/_internal/commands/uninstall.py

```diff
@@ -3,6 +3,7 @@
 from pip._internal.cli.base_command import Command
 from pip._internal.cli.req_command import SessionCommandMixin
 from pip._internal.cli.status_codes import SUCCESS
+from pip._internal.commands.install import prepare_sudo_warning
```

Comment:
I'd much rather put this in a different location. I don't want to create a dependency between two commands like this.

Please move the helper function into cli.req_command instead.